src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/X86/Aes.cs

```diff
@@ -11,10 +11,10 @@ namespace System.Runtime.Intrinsics.X86
     /// This class provides access to Intel AES hardware instructions via intrinsics
     /// </summary>
     [CLSCompliant(false)]
-    public static class Aes 
+    public abstract class Aes : Sse2
     {
-        public static bool IsSupported { get => IsSupported; }
-        
+        new public static bool IsSupported { get => IsSupported; }
```

Comment:
I believe `public new` is the "typical" order.  See [https://github.com/dotnet/machinelearning/pull/557#discussion_r204118747](https://github.com/dotnet/machinelearning/pull/557#discussion_r204118747) for the same discussion from last week in dotnet/machinelearning.

Reply:
Fixed to match the IDE default.